jiant/models.py

```diff
@@ -1094,7 +1113,11 @@ def _lm_forward(self, batch, task, predict):
         pad_idx = self.vocab.get_token_index(self.vocab._padding_token, "tokens")
         b_size, seq_len = batch["targs"]["words"].size()
         n_pad = batch["targs"]["words"].eq(pad_idx).sum().item()
-        out["n_exs"] = (b_size * seq_len - n_pad) * 2
+        out["n_exs"] = (
```

Comment:
slight preference to just split this in two lines rather than have the redundancy

Reply:
what do you mean split into two lines? Even if we split the if else into two lines it'll still be redundant to those few lines. I thought of moving it out of these task specific forward functions but different tasks have different ways of calculating n_exs so that wouldn't work.